thermostat_api_server.py

```diff
@@ -397,7 +398,8 @@ class ThreadingSimpleServer(ThreadingMixIn, HTTPServer):
     pass
 
 client = mqttClient.Client(f"thermostat_api_server_{thermostat_serial}")
-client.username_pw_set(username=mqtt_username,password=mqtt_password)
+if (mqtt_username):
```

Comment:
Same here. This should do the intended:
`if "mqtt_username" in locals():`

Reply:
Fixed this too.